README.md

```diff
@@ -14,12 +14,12 @@ Project Objectives
 - Provide OpenAPI description writers for both V2 and V3 specification formats.
 - Enable developers to create Readers that translate different data formats into OpenAPI descriptions. 
 
-# Readers
-The OpenAPI.NET project holds the base object model for representing OpenAPI descriptions as .NET objects. Translation for different data types into this object model is handled by reading raw JSON/YAML or from individual "Readers", a number of which are in the works.
+# Processors

```

Comment:
>Processors [](start = 2, length = 10)

Should we call it Generators, to be in line with term that we use in Annotations repo?